src/index.html

```diff
@@ -1,17 +1,209 @@
+    <script src="./main.js"></script>
+    <form 
+      class="form" 
+      action="http://localhost:5500/api" 
+      method="post"
+    >
+      <fieldset>
+        <legend class="legend">Personal Information</legend>
+        <label class="form-field">
+          Surname:
+          <input 
+            class="input"
+            type="text" 
+            name="surname" 
+            required 
+            pattern="\w+"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-field">
+          Name:
+          <input 
+            class="input"
+            type="text" 
+            name="name" 
+            required 
+            pattern="\w+"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-field">
+          How old are you?
+          <input 
+            class="input"
+            type="number" 
+            name="age" 
+            min="1"
+            max="100"
+            required 
+            placeholder="12"
+          >
+        </label>
+        <label class="form-field">
+          Full date of birth:
+          <input 
+            class="input"
+            type="date" 
+            name="dob" 
+            required 
+            pattern="\w+"
+          >
+        </label>
+        <label class="form-field">
+          <label 
+            class="label"
+            for="terms-and-conditions"
+          >I accept the terms of the agreement:</label>
+          <input
+            class="input" 
+            type="checkbox" 
+            name="terms" 
+            id="terms-and-conditions" 
+            required
+          >
+        </label>
+      </fieldset>
+    </form>
+    <form 
+      class="form" 
+      action="http://localhost:5500/api" 
+      method="post"
+    >
+      <fieldset>
+        <legend class="legend">Registration</legend>
+        <label class="form-field">
+            Email:
+          <input 
+            class="input"
+            type="email" 
+            name="email" 
+            placeholder="email@example.com" 
+            required
+          >
+        </label>
+        <label class="form-field">
+          Password:
+          <input 
+            class="input"
+            type="password" 
+            name="password" 
+            minlength="5" 
+            maxlength="30" 
+            pattern="\d+"
+            required
+          >
+        </label>
+      </fieldset>
+    </form>
+      
+    <form 
+      class="form" 
+      action="http://localhost:5500/api" 
+      method="post"
+    >
+      <fieldset>
+        <legend class="legend">An Interesting Fact About You!</legend>
+        <label>
+          Do you love cats?
+          <input 
+            class="input"
+            type="radio" 
+            id="radioButton1" 
+            name="cats"
+          >
+          <label 
+            class="label" 
+            for="radioButton1"
+          >Yes</label>
```

Comment:
please, check code style for other labels as well

```suggestion
          <label 
            class="label" 
            for="radioButton1"
          >
            Yes
         </label>
```